packages/tools/pie-icons-configs/configs.js

```diff
@@ -56,17 +56,48 @@ export function validateGetLargeIconSize (iconSizeValue) {
 /**
  * Validates the iconSize for regular icons
  * @param {string} iconSizeValue - Value of the iconSize prop
- * @returns {{isValid: boolean, iconSizeClass: string}} - Object with the validation result and the icon size class
+ * @returns {{isValid: boolean, iconSize: number}} - Object with the validation result and the icon size
  */
-export function validateGetRegularIconClass (iconSizeValue) {
+export function validateGetRegularIconSize (iconSizeValue) {
     const isValid = iconSizeValidator.regular(iconSizeValue);
 
     if (!isValid) {
-        const iconSizeClass = sizeToClassMap[regularIconSizeDefault];
-        return { isValid: false, iconSizeClass };
+        const iconSize = sizeToValueMap[regularIconSizeDefault];
+        return { isValid: false, iconSize };
     }
 
-    const iconSizeClass = sizeToClassMap[iconSizeValue];
+    const iconSize = sizeToValueMap[iconSizeValue];
 
-    return { isValid: true, iconSizeClass };
+    return { isValid: true, iconSize };
 }
+
+/**
+ * Returns props for the svg element based on the provided parameters
+ * @param {string} svgClasses - String of classes assigned to the icon
+ * @param {string} staticClass - String of classes assigned to the component
```

Comment:
Small one – should this be `staticClasses`? Can't work out if this could potentially be multiple or not 😆

Reply:
Yes, it can be a string of multiple class names 👍